domains/transfer/Transfer.API/API.IntegrationTests/SwaggerTests.GetSwaggerDocs_v20230101.verified.txt

```diff
@@ -873,14 +1064,16 @@
             "format": "uuid"
           },
           "companyATin": {
-            "type": "string"
+            "type": "string",
+            "nullable": true
```

Comment:
There is a lot of changes to nullable: true. Why is this changed?

Reply:
Hmm thats a good question. Looking into it.

Reply:
options.SupportNonNullableReferenceTypes(); was accidently removed. 
fixed in 63ba39a2